app/assets/javascripts/helpers/SerializeFilters.js

```diff
@@ -0,0 +1,37 @@
+(function (App) {
+  'use strict';
+
+  App.Helper.SerializeFilters = function (form) {
```

Comment:
I think I would have that code into a helper. My definition of an helper is that it's meant to be a utility function that you could reuse somewhere else in your app.

In your case, it seems it's quite tied to the filters.